packages/expo/src/Linking/LinkingModule.web.ts

```diff
@@ -0,0 +1,40 @@
+import NativeEventEmitter from 'react-native/Libraries/EventEmitter/NativeEventEmitter';
+import invariant from 'invariant';
+
+class RNLinking extends NativeEventEmitter {
```

Comment:
It might make more sense not to extend NativeEventEmitter since there's no "native" (that is, web) implementation of LinkingManager. `expo-core`'s NativeEventEmitter's job is to provide convenient add/removeListener methods that are forwarded to the native module (and on Android it calls start/stopObserving when adding/removing the first/last listener), but all of that isn't necessary here since we're providing an entire implementation of Linking for the web, as opposed to following our typical pattern of implementing just the "native" module for web.